### PR TITLE
Add a minimum image

### PR DESCRIPTION
**Pull request by bells17**
resolved: #196

~~I added a minimum image(this is an image that excluded CSI sidecars).
This image has a "minumum-xxx" tag.
(I didn't change a tag of a non-minimum image because it needs changing tags of  all manifest files if changed a tag of a non-minimum image)~~

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -28,11 +28,6 @@ RUN ln -s hypertopolvm /lvmd \
     && ln -s hypertopolvm /topolvm-controller
 
 # CSI sidecar
```
Comment (toshipp) [inline Dockerfile:30]: This is not needed.

**Review by toshipp — CHANGES_REQUESTED** [review]

**Comment by llamerada-jp** [comment]
On a side note would you update the URL to quay.io in the bottom of README.md ?
New URL is `https://quay.io/organization/topolvm`

File: Dockerfile (hunk 1 above)
Comment (bells17) [reply Dockerfile]: Thanks. I removed it.

**Comment by bells17** [comment]
https://github.com/topolvm/topolvm/pull/236#issuecomment-760017458
I updated the links.

**Comment by bells17** [comment]
@toshipp @llamerada-jp Hi, I updated. Could you review this again?

**Review by toshipp — APPROVED** [review]
LGTM

File: Dockerfile.with-sidecar [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# Build Container
+FROM golang:1.13-buster AS build-env
```
Comment (llamerada-jp) [inline Dockerfile.with-sidecar:2]: Please use golang 1.15 as the PR that updates golang has been merged recently.

https://github.com/topolvm/topolvm/blob/adcd4c9fd4ba26e24d1ee130a0e28646350ce7ab/Dockerfile#L2

File: Dockerfile.with-sidecar [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+# Get argment
+ARG TOPOLVM_VERSION
+
+COPY . /workdir
+WORKDIR /workdir
+
+RUN make build TOPOLVM_VERSION=${TOPOLVM_VERSION}
+
+# TopoLVM container
+FROM ubuntu:18.04
```
Comment (llamerada-jp) [inline Dockerfile.with-sidecar:13]: Please use ubuntu 20.04 as the PR that updates ubuntu has been merged recently.

https://github.com/topolvm/topolvm/blob/adcd4c9fd4ba26e24d1ee130a0e28646350ce7ab/Dockerfile#L13

**Review by llamerada-jp — CHANGES_REQUESTED** [review]

**Review by toshipp — APPROVED** [review]
LGTM

**Review by llamerada-jp — APPROVED** [review]
LGTM

**Pull request merged**